GPU/Common/IndexGenerator.cpp

```diff
@@ -174,7 +174,7 @@ void IndexGenerator::AddLineStrip(int numVerts) {
 void IndexGenerator::AddRectangles(int numVerts) {
 	u16 *outInds = inds_;
 	const int startIndex = index_;
-	for (int i = 0; i < numVerts; i += 2) {
+	for (int i = 0; i +1 < numVerts; i += 2) {
```

Comment:
I wonder if it matters that `count_` increases by `numInds` and `outInds` doesn't.... maybe need to `& ~1` it also... (and below.)

Also, here there's no space before the `1` in `+1`.

-[Unknown]